packages/autocomplete/src/commands/autocomplete/options.ts

```diff
@@ -103,12 +103,14 @@ export default class Options extends AutocompleteBase {
 
   private async fetchOptions(cache: any) {
     const {cacheCompletion, cacheKey} = cache
+    const flags = await this.parsedFlagsWithEnvVars()
```

Comment:
Why did we need to `await` here?

Reply:
One thing that changed with oclif/core is that they made the parse methods asynchronous - so it's necessary to await.